src/Modal/Modal.js

```diff
@@ -34,27 +34,33 @@ class Modal extends Component {
     render() {
         const {
             actions,
+            backdropClassName,
             bodyProps,
             children,
-            className,
             closeProps,
             contentProps,
             disableStyles,
             headerProps,
             headingLevel,
             footerProps,
             localizedText,
+            modalClassName,
             onClose,
             show,
             titleProps,
             title,
             ...rest
         } = this.props;
 
-        const modalClasses = classnames(
+        const backdropClasses = classnames(
             'fd-overlay',
             'fd-overlay--modal',
-            className
+            backdropClassName
+        );
+
+        const modalClasses = classnames(
+            'modal-demo-bg',
+            modalClassName
         );
```

Comment:
Are all these changes correct?  Maybe this didn't work before, but I'm seeing a mismatch in what classes were added to which elements compared to before.

Looking below, there were `modalClasses` being added to the `<FocusLock>` component which previously would have been `fd-overlay` and such and now it would be `modal-demo-bg` and such.  There are additional mismatches too.